config/software/libtool.rb

```diff
@@ -16,31 +15,36 @@
 #
 
 name "libtool"
-default_version "2.4.2"
+default_version "2.4"
 
-version "2.4" do
-  source md5: "b32b04148ecdd7344abc6fe8bd1bb021"
-end
+license "GPL-2.0"
+license_file "COPYING"
+skip_transitive_dependency_licensing true
 
-version "2.4.2" do
-  source md5: "d2f3b7d4627e69e13514a40e72a24d50"
-end
+dependency "config_guess"
 
-source url: "http://ftp.gnu.org/gnu/libtool/libtool-#{version}.tar.gz",
-       extract: :seven_zip
+# NOTE: 2.4.6 2.4.2 do not compile on solaris2 yet
+version("2.4.6") { source md5: "addf44b646ddb4e3919805aa88fa7c5e" }
+version("2.4.2") { source md5: "d2f3b7d4627e69e13514a40e72a24d50" }
+version("2.4")   { source md5: "b32b04148ecdd7344abc6fe8bd1bb021" }
+
+source url: "https://ftp.gnu.org/gnu/libtool/libtool-#{version}.tar.gz"
 
 relative_path "libtool-#{version}"
-env = with_embedded_path()
-# AIX uses gcc/g++ instead of xlc/xlC
-env = with_standard_compiler_flags(env, :aix => { :use_gcc => true })
 
 build do
-  mkdir "/tmp/build/embedded"
-  if ohai["platform"] == "aix"
-    command "./configure --prefix=/tmp/build/embedded --with-gcc", :env => env
```

Comment:
Can we still avoid actually shipping `libtool` in the final pkg? See https://github.com/DataDog/omnibus-software/commit/694760a63d6416602c7cad8919af0941c612211c